README.md

```diff
@@ -1 +1,7 @@
-# steampipe-cloud-zapier
\ No newline at end of file
+# Steampipe Cloud Zapier Integration
+
+[Zapier](https://zapier.com/) is a web-based platform for creating automated, no-code workflows. Zapier interacts with other applications and services via Integrations.
+
+Steampipe Cloud integration will allow the Steampipe Cloud users to leverage their Steampipe Cloud Workspaces in their Zapier workflows.
+
+If you have any questions, join our [Slack Community](https://steampipe.io/community/join) and ask away!
```

Comment:
Can you please add a section here on how to publish a new version, e.g., log in with CLI, update package.json, run commands?